lib/msf/base/sessions/ssh_command_shell_bind.rb

```diff
@@ -153,6 +153,73 @@ def write(buf, length = nil)
       attr_reader :cid, :client, :params
     end
 
+    # Represents an SSH reverse port forward.
+    # Will receive connection messages back from the SSH server,
+    # whereupon a TcpClientChannel will be opened
+    class TcpServerChannel
+      def initialize(params, client, host, port)
```

Comment:
This should track a channel ID as well, like the ClientChannel does.
```suggestion
      def initialize(client, cid, params, host, port)
```

Reply:
As above - I presume channel IDs should be used for client channels, as that relates to the #channels attribute.